test/async-hooks/test-graph.tls-write.js

```diff
@@ -20,8 +20,8 @@ hooks.enable();
 //
 const server = tls
   .createServer({
-    cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
-    key: fs.readFileSync(common.fixturesDir + '/test_key.pem')
+    cert: fs.readFileSync(`${common.fixturesDir}/test_cert.pem`),
```

Comment:
We have had the same discussion in other PRs before, and we would have to change this in quite many files. It does not make it worse than it is :)